src/copyediting/migrations/0005_auto_20190523_1758.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.20 on 2019-05-23 17:58
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def replace_author_review_setting(apps, schema_editor):
+    Setting = apps.get_model('core', 'Setting')
+    SettingValue = apps.get_model('core', 'SettingValue')
+    SettingValueTranslation = apps.get_model('core', 'SettingValueTranslation')
+    Journal = apps.get_model('journal', 'Journal')
+
+    setting = Setting.objects.filter(
+        name='copyeditor_notify_author',
+        group__name='email',
+    )
+
+    journals = Journal.objects.all()
+
+    for journal in journals:
+        values = SettingValue.objects.filter(journal=journal, setting=setting)
+
+        for value in values:
+            translations = SettingValueTranslation.objects.filter(
+                master_id=value.pk
+            )
+
+            for translation in translations:
+                value = translation.value
+                translation.value = value.replace(
+                    '{{journal.site_url}}{{ url }}',
```

Comment:
the default value that you replaced below was:
`{{request.journal.site_url}}{{ url }}`
We need to handle that as well

Reply:
Added.